# A list of float values with timestamp

Header header
float64[] data